src/components/Card/CardContent/CardContent.js

```diff
@@ -1,30 +1,47 @@
 import { useState } from "react";
 import { useLocation } from "react-router-dom";
 import { getElapsedTime } from "../../../utils/getElapsedTime";
+import { formatDate } from "../../../utils/formatDate";
+import { useModal } from "../../../hooks/useModal";
+import { DeleteLinkModal, AddLinkModal } from "../../Modal";
 import "./CardContent.css";
 
-function formatDate(value) {
-  const date = new Date(value);
-  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
-}
-
-export const CardContent = ({ items, isZoomedIn, handleLinkClick }) => {
+export const CardContent = ({
+  items,
+  isZoomedIn,
+  handlePreventLinkClick,
+  linkUrl,
+  folderList,
+}) => {
   const [showOptions, setShowOptions] = useState(false);
 
+  const {
+    openModal: deleteOpenModal,
+    modalRef: deleteModalRef,
+    handleModalClose: deleteHandleModalClose,
+    handleModalOpen: deleteHandleModalOpen,
```

Comment:
P2:
함수인지 알 수 잇는 이름이면 좋겠어요.

```suggestion
    handleModalClose: handleDeleteModalClose,
    handleModalOpen: handleDeleteModalOpen,
```